lib/config.js

```diff
@@ -19,5 +19,18 @@ module.exports = function() {
     }
   };
 
-  return rc('couch_email_auth', defaultConfig);
+  var config = rc('couch_email_auth', defaultConfig);
+
+  if (typeof config.allowedRedirectHosts === 'string' &&
+      config.allowedRedirectHosts.trim() !== '') {
+    config.allowedRedirectHosts = config.allowedRedirectHosts.split(',').map(
+      function(host) {
+        return host.trim();
+      }
+    );
+  } else {
```

Comment:
Do you have a concrete suggestion?

I'm +1 on early returns where they result in code paths that are easier to follow. But here I want to make sure that `config.allowedRedirectHosts` is always set and and of type array, and I don't see how to do an early return without making it less understandable.